tests/address_test.go

```diff
@@ -74,6 +74,34 @@ func (c *ClientTests) TestAddressAll() {
 	}
 }
 
+func (c *ClientTests) TestAddressGetNextPage() {
+	client := c.TestClient()
+	assert, require := c.Assert(), c.Require()
+
+	firstPage, err := client.ListAddresses(
+		&easypost.ListOptions{
+			PageSize: c.fixture.pageSize(),
+		},
+	)
+	require.NoError(err)
+
+	nextPage, err := client.GetNextAddressPageWithPageSize(firstPage, c.fixture.pageSize())
+	defer func() {
```

Comment:
what is the reason for this `defer()` func? why not place the contents of this function at the end of the outer func?

Reply:
Hmm, good point now that I look at it. Was trying to handle the equivalent of a "try-catch" in Go, but looks like I might have wired it redundantly.